bundle.md

```diff
@@ -14,7 +16,7 @@ A standard container bundle is made of the following 3 parts:
 - One or more content directories
 - A configuration file
 
-# Directory layout
+## Directory layout
```

Comment:
This seems like a subsection of the previous section (e.g. the previous section has the list of the three allowed parts).  So maybe remove the `## Container Format` header?